src/test/java/org/hiero/sketch/MembershipTest.java

```diff
@@ -35,13 +49,11 @@ public void TestPartialDense() {
         assert(PMD.isMember(6));
```

Comment:
use assertEquals.
In fact I fixed this in my own pull request, but I haven't merged yet (and I have fixed other things in your tests). Take a look at my code.
In general, don't print in tests - it's useless when you have 1000 tests. Just use assertions.